react/PartnerSites/PartnerSites.js

```diff
@@ -0,0 +1,35 @@
+import styles from './PartnerSites.less';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Hidden from '../Hidden/Hidden';
+import PageBlock from '../PageBlock/PageBlock';
+import Locales from './Locales/Locales';
+import Sites from './Sites/Sites';
+
+const defaultLinkRenderer = props => (<a {...props} />);
+
+export default function PartnerSites({ locale, linkRenderer, activeSite }) {
+  return (<Hidden component={PageBlock} print mobile className={styles.root}>
+    <div className={styles.content}>
+      <Sites locale={locale} linkRenderer={linkRenderer} activeSite={activeSite} />
+      <div className={styles.locale}>
+        <Locales locale={locale} linkRenderer={linkRenderer} />
+      </div>
+    </div>
+  </Hidden>);
+}
+
+PartnerSites.displayName = 'PartnerSites';
+
+PartnerSites.propTypes = {
```

Comment:
shall we flow type "new" components?

Reply:
@kosanna I've not used flow before and have already spent a considerable amount of time on this PR in between project work. Is this something that could be picked up as a separate PR or would this block this PR from being merged?

Reply:
go for it, we'll address it separately together 👍

Reply:
Thanks @kosanna. That sounds good to me 👍